src/mmw/apps/modeling/views.py

```diff
@@ -539,6 +540,27 @@ def get_layer_shape(table_code, id):
         else:
             return None
 
+def parse_area_of_interest(area_of_interest, geojson=True):
```

Comment:
This function could use some comments about the arguments.  It's not clear why there's a flag for `geoJson` but the function raises an error if there is invalid geojson provided, despite the flag.  If we have two workflows that use this function with different types, it may be worth modifying the calls so they are both passing a single type.  Goes for the related function below.

Reply:
I'll add comment to these two and the `parse_input` in the `geoprocessing_api`.

> It's not clear why there's a flag for geoJson but the function raises an error if there is invalid geojson provided, despite the flag.

The function always expects `area_of_interest` to be valid geojson. The `geojson` flag controls whether the function returns a geojson string or its `dict` representation. It comes from the original `modeling.views.parse_input` implementation; I'll see if we can take it out.